README.md

```diff
@@ -65,84 +74,104 @@ If you don't use Google PubSub, sett this property:
     spring.cloud.gcp.pubsub.enabled=false
 
 ## Running locally
-### Build
-To build the project from source, you need Java 21 and Maven 3.
 
-To run the unit tests you need additionally to install the [Google PubSub emulator](https://cloud.google.com/pubsub/docs/emulator).  
-The emulator can be installed locally with the following commands:
-```
-gcloud components install pubsub-emulator
-gcloud components update
-```
-You can then locate the installation directory of the GCloud tool kit with the following command:
+### Build
 
-```
-gcloud info --format="value(installation.sdk_root)"
-```
-The emulator executable is located under the sub-folder ```platform/pubsub-emulator/lib```  
-and can be used in unit tests by setting the following property in the Spring Boot properties file:
-```
-entur.pubsub.emulator.path=/usr/lib/google-cloud-sdk/platform/pubsub-emulator/lib/cloud-pubsub-emulator-0.6.0.jar
-```
+To build the project from source, you need Java 21 and Maven 3.
 
 ### Database
+
 #### Via Docker
 
 Install Docker.
 
-```bash
-docker run --name=uttu -d -e POSTGRES_USER=uttu -e POSTGRES_PASSWORD=uttu -e POSTGRES_DB=uttu -p 5432:5432 -v db_local:/var/lib/postgresql --restart=always postgis/postgis:13-3.3
+```shell
+docker run --platform linux/amd64 --name=uttu -d -e POSTGRES_USER=uttu -e POSTGRES_PASSWORD=uttu -e POSTGRES_DB=uttu -p 
+5432:5432 -v db_local:/var/lib/postgresql --restart=always postgis/postgis:13-3.3
 ```
 
 Now a Docker container is running in the background. Check its status with `docker ps`.
 
-To stop, find its ID from `docker ps`, and run `docker stop theid` (beginning of hash). To restart it, find the ID from `docker container list` and run `docker restart theid`.
+To stop, find its ID from `docker ps`, and run `docker stop theid` (beginning of hash). To restart it, find the ID from 
+`docker container list` and run `docker restart theid`.
+
+Run the [database initialization script](./src/main/resources/db_init.sh).
 
-Run the [script](./src/main/resources/db_init.sh).
+Run the [database init script](./src/main/resources/db_init.sh).
```

Comment:
Seems to be repetition if L98, so remove either this or that.

Reply:
This was an accidental rebase artifact. Fixed.